Formula/portable-openssl.rb

```diff
@@ -39,6 +39,7 @@ def arch_args
       {
         :x86_64 => %w[linux-x86_64],
         :i386  => %w[linux-generic32],
+        :arm => %w[linux-armv4],
```

Comment:
Should this be `armv6` for Raspberry Pi version 1? What's the minimum ARM CPU that you plan to support?